deps/build.jl

```diff
@@ -73,30 +74,44 @@ mkpath(lib_dir)
 mkpath(bin_dir)
 
 
-function download_and_unpack(url)
-    tensorflow_zip_path = joinpath(base, "downloads/tensorflow.tar.gz")
-    download(url, tensorflow_zip_path)
+function download_and_unpack(url, checksum)
+    tensorflow_zip_path = joinpath(download_dir, "tensorflow.tar.gz")
+
+    use_cache = false
+    isfile(tensorflow_zip_path) && open(tensorflow_zip_path) do f
+        use_cache = bytes2hex(sha256(f)) == checksum
+    end
+        
+    if use_cache
+        @info "using cached $tensorflow_zip_path"
+    else
+        @info "downloading tensorflow from $url"
+        download(url, tensorflow_zip_path)
+    end
     run(`tar -xzf $tensorflow_zip_path -C downloads`)
 end
 
 @static if Sys.isapple()
     if use_gpu
```

Comment:
should I delete this `if use_gpu` block? Since https://github.com/malmaud/TensorFlow.jl/blob/4c43fa2194f77c35c88110bb48228a2e19020b76/deps/build.jl#L25 already checked.